src/BuildScriptGeneratorCli/Commands/CommandBase.cs

```diff
@@ -26,6 +28,12 @@ internal abstract class CommandBase
         [Option("--debug", Description = "Print stack traces for exceptions.")]
         public bool DebugMode { get; set; }
 
+        [Option("--github-build-start", Description = "The time when a Github Action started the build.")]
+        public string GitHubBuildStartTime { get; set; }
```

Comment:
@qianz2 I'm in the boat that these GitHub Action-specific properties we look for should be set as environment variables during the action -- this will keep us from introducing new parameters each time there's another property of an action we want to capture and use in some fashion, and potentially "cluttering" the set of parameters we have for our CLI.

What do you think? CC: @kichalla @arroyc

Reply:
👍 for environment variables

Reply:
I vote for environment variable too